confgenerator/testdata/invalid/linux/metrics-processor_exclude_metrics_invalid_parameter_metric_prefixes/golden_error

```diff
@@ -1 +1,9 @@
-parameter "metrics_pattern" in "exclude_metrics" type metrics processor "metrics_filter" has invalid value ["agent.googleapis.com/processes/not_end_with_forward_slash_with_star" "not.agent.googleapis.com/processes/*"]: "agent.googleapis.com/processes/not_end_with_forward_slash_with_star" must end with "/*" | "not.agent.googleapis.com/processes/*" must start with "agent.googleapis.com/"
\ No newline at end of file
+the agent config file is not valid. detailed error: [6:7] Key: 'MetricsProcessorExcludeMetrics.metrics_pattern[0]' Error:Field validation for 'metrics_pattern[0]' failed on the 'endswith' tag,Key: 'MetricsProcessorExcludeMetrics.metrics_pattern[1]' Error:Field validation for 'metrics_pattern[1]' failed on the 'startswith' tag
```

Comment:
Might need to break this one into 2 test cases.

Reply:
Meh, we already have too many tests :P

This will hopefully be better once we improve the error message.